tests/unit/adapters/application-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
```

Comment:
Probably won't be testing this unless you're adding actual content to the application adapter. File can be deleted.